### PR TITLE
Fix display of dropped questions tooltip on hw score

### PR DESCRIPTION
**Pull request by nathanstitt**
image

image

### CODE REVIEW

File: tutor/src/components/dropped-question.tsx [hunk 1]
```diff
@@ -84,16 +85,15 @@ const DroppedQuestionHeadingIndicator: React.FC<DroppedQuestionHeadingIndicatorP
 }) => {
     if (!heading.someQuestionsDropped) return null
     let tooltip
+    const students = pluralize('student', heading.droppedQuestions.length, true)
     if (heading.isCore) {
```
Comment (Dantemss) [inline tutor/src/components/dropped-question.tsx:89]: I think this is also changing it for homework core which we do not want.

File: tutor/src/components/dropped-question.tsx [hunk 2]
```diff
@@ -108,9 +108,39 @@ const DroppedQuestionHeadingIndicator: React.FC<DroppedQuestionHeadingIndicatorP
...
+    }
+}
+export const DroppedReviewExerciseIndicator: React.FC<DroppedReviewExerciseIndicatorProps> = observer(({
+    info,
+}) => {
+    const model = info.droppedQuestion
+
+    if (!model || !model.drop_method) return null
+
+    let students = pluralize('student', info.responses.length, true)
```
Comment (Dantemss) [inline tutor/src/components/dropped-question.tsx:127]: Same thing here, the change from "all" to a number will also affect homework core which we do not want.

File: tutor/src/screens/assignment-review/homework-scores.js [hunk 3]
```diff
@@ -235,7 +235,7 @@ const TaskResult = observer(({ heading, result, striped }) => {
             isTrouble={result && !result.isUnattemptedAutoZero && result.isTrouble}
             isUnattemptedAutoZero={result && result.isUnattemptedAutoZero}
         >
-            {!heading.isCore && <DroppedQuestionHeadingIndicator heading={heading} />}
```
Comment (Dantemss) [inline tutor/src/screens/assignment-review/homework-scores.js:238]: I believe this triangle should only appear if the one on https://github.com/openstax/tutor-js/blob/657effafe37bf87e32857010565f70f5356e1f81/tutor/src/screens/assignment-review/homework-scores.js#L83 does not so we need the !isCore check.

**Review by Dantemss — CHANGES_REQUESTED** [review]
I think rather than removing the isCore checks we need to add some checks for reading vs homework.

**Comment by nathanstitt** [comment]
yes it is change it for all HW core, but we do want that as I understand it.  I think we want to show the count at all times.  lets PL

**Review by Dantemss — APPROVED** [review]
Looks good to me and Devika said OK to the change from `Question dropped for one or more students` to `Question dropped for # students`

**Review by Dantemss — CHANGES_REQUESTED** [review]
Sorry one more thing... the student count in whatever replaces DroppedReviewExerciseIndicator has to come from info, not the heading I believe. The heading aggregates in ways it shouldn't for DroppedReviewExerciseIndicator so the count will not be right

**Review by Dantemss — APPROVED** [review]

**Pull request merged**